frontend/src/pages/checklist/[filters].tsx

```diff
@@ -151,6 +151,7 @@ const ChecklistResults: FC<ChecklistResultsProps> = ({
       >
         <div className="hidden md:block">
           <HeroBanner
+            className='h-[240px] md:min-h-[240px] grid-rows-1'
```

Comment:
I think you can safely remove `h-[240px]` and instead do `min-h-[240px] grid-rows-1'`.

Reply:
Actually might be better to remove all the min heights all together and just inject a custom height once? Seems like the min-heights are having no effect on the sizing in any viewport.

Reply:
possibly because the image exceeds a height of 240px by default?  Not sure really.  I think this is ready for approval though.